user-guide/Advanced_Functionality/Swarming/Tutorials/EnableSwarmingTutorial.md

```diff
@@ -0,0 +1,156 @@
+   The AlarmID usage refers to the unique identifier of alarms and their usage in scripting (Automation/QActions).
+
+   ![AlarmID Usage](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage.png)
+
+   In order for Swarming to work, this alarm identifier has been updated which is a breaking change.
+   To prevent possible issues as much as possible, there is a check to analyze your scripts automatically for problems.
+   This is not perfect and does not scan everything but will already catch some obvious problems.
+
+   Press the **Analyze** and wait for a bit until the results appear. This can take a while, up to several minutes depending on how many scripts and protocols you have.
+
+   ![AlarmID Usage Problem](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem.png)
+
+   After analysis is done, the **Mask Critical Alarms** script should appear as one of the found problems.
+   This script takes as input a reference to an element and will mask all critical alarms of this element.
+   It does so by using a **SetAlarmStateMessage** and includes an identifier reference to the alarm it should mask.
+   This identifier still uses the old style of **DataMinerID/AlarmID** integer combination. This is no longer correct and will need a **TreeID** instead.
+   If you were to open this script via DIS in Visual Studio, you would see an obsolete warning.
+   You can also see this if you open the script via the Automation app and press the **Validate** button to check for errors and warnings.
+
+   ![AlarmID Usage Problem Warning](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem_Warning.png)
+
+   Update the code in the foreach block (Line 85) as follows:
+
+    Before:
+
+    ```csharp
+        var maskRequest = new SetAlarmStateMessage();
+        maskRequest.DataMinerID = alarm.DataMinerID;
+        maskRequest.AlarmId = alarm.AlarmID;
+        maskRequest.DesiredStatus = AlarmUserStatus.Mask;
+        maskRequests.Add(maskRequest);
+    ```
+
+    After:
+
+    ```csharp
+        var maskRequest = new SetAlarmStateMessage(alarm.TreeID, AlarmUserStatus.Mask, "");
+        maskRequests.Add(maskRequest);
+    ```
+
+   Press the **validate** button to see if all warnings and errors are gone.
+
+   ![AlarmID Usage Problem Fix](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem_Fix.png)
+
+   If any other scripts or protocols appear, this will also need an update.
+   There will always be an overload or constructor you can use instead, which will refer to the **TreeID** or **AlarmTreeID**.
+
+   Please verify any updated script or protocol still works.
+
+1. Enabling Swarming
+
+   Once the static requirements are met and there are no AlarmID usage problems detected. A button will appear that you can enable Swarming.
+   Pressing this button will send the enable command to your DMS and will **restart the whole DMS**. Afterwards, Swarming will be enabled.
```

Comment:
maybe good to mention that the checks will be executed again, so this action can take a while (I know the screenshot also has this, but maybe good to have it in text as well)

Reply:
Done